onedal/basic_statistics/basic_statistics.py

```diff
@@ -19,13 +19,16 @@
 
 import numpy as np
 
-from ..common._base import BaseEstimator
+from .._backend import basic_statistics as onedal_backend
+from .._base import BaseEstimator
 from ..datatypes import _convert_to_supported, from_table, to_table
 from ..utils import _is_csr
 from ..utils.validation import _check_array
 
 
 class BaseBasicStatistics(BaseEstimator, metaclass=ABCMeta):
+    _backend = onedal_backend
```

Comment:
If we already defined _backend in BaseEstimator, why it is needed here?

Reply:
@samir-nasibli  ```_backend``` in this case is ```onedal._backend.basic_statistics```.  In a future PR we can assign the necessary methods to the estimators as functions, and overwrite them in the spmd estimators in a case-by-case basis.  That is a lot of work to switch this in this PR, but the correct one for clarity and extensibility. The changes here are an intermediate step. Otherwise this PR may be a 500+ line change.